contents/blog/google-analytics-to-posthog.md

```diff
@@ -0,0 +1,135 @@
+
+- Every event contains a distinct ID for a person.
+- With the snippet, every person gets an anonymous distinct ID. This gets stored in a cookie and automatically added to the events they trigger.
+- You can use the [identify](/docs/data/identify) function to connect a person with a distinct ID like email or username.
+- You can use the [group](/docs/product-analytics/group-analytics) function to connect a person with a group like an organization or company.
+
+![Event structure](../images/blog/google-analytics-to-posthog/event.png)
+
+You can then use all of this data in your insights. For example, you can aggregate an event series by unique users, unique groups, monthly active groups, and more. 
+
+## Filtering with dimensions (aka properties)
+
+Like Google Analytics, each of the event series, insights, and dashboards is filterable. Filters are largely created with our version of "dimensions," named [properties](/docs/data/events#event-properties). 
+
+Properties are keys and values set on events, persons, and groups adding more details about them. For example, the pageview event contains properties for the timestamp, OS, browser, current URL, referring domain, geographic data, and more. PostHog automatically adds some properties, but if you capture [custom events](/docs/libraries/js#send-custom-events-with-posthogcapture), you can also send as many custom properties as you want.
+
+```js
+posthog.capture('signed up', {email: 'ian@posthog.com', paid: true});
+```
+
+You can also use properties to create [cohorts](/docs/data/cohorts). These are lists of users meeting behavior or property criteria, such as completing an event for the first time or being a paid user. You can then use cohorts as filters throughout PostHog. 
+
+## Goals and conversions (aka actions)
+
+The PostHog equivalent to goals and conversions are [actions](/docs/data/actions). An action is one or more events with filters matching a user’s behavior. For example, a "shown interest" action could include a visit to the pricing page, clicking a book demo button, or signing up.
+
+You create actions in the [data management tab](https://app.posthog.com/data-management/actions). Once created, you can use them in insights. For example, if you want to track conversion, they are especially useful in the funnel-type insight. Actions can also trigger [webhooks](/docs/webhooks) and send messages in [Slack](/docs/webhooks/slack) or [Teams](/docs/webhooks/microsoft-teams).
+
+Beyond actions, PostHog also can run A/B tests, which compare "test" and "control" variants to calculate whether a change impacts a goal metric. This is our version of Google Optimize. For an example, see "[How to run A/B tests in Webflow with PostHog](/tutorials/webflow-ab-tests)."
+
```

Comment:
I think this would be a good place to add some kind of "top tips for n00bs" type section. I'm happy to pick this up if you like?

Reply:
Sure, that would be great